docs/core/sdk.md

```diff
@@ -22,7 +22,7 @@ It contains the following components:
 
 1. The .NET Core Command Line Tools that are used to build applications
 2. .NET Core (libraries and runtime) that allow applications to both be built and run
-3. The `dotnet` driver for running the CLI commands as well as running applications
+3. The `dotnet` driver for running the [CLI commands](tools/index.md) as well as running applications
 
 
 ## Acquiring the .NET Core SDK
```

Comment:
Maybe adding what's the meaning of these labels will be helpful for contributors.